docs/docs/index.md

```diff
@@ -4,31 +4,32 @@ description: The one stop location for tutorials, guides, and information about
 disableTableOfContents: true
 ---
 
-Gatsby is a blazing fast modern site generator for React.
+Gatsby React tabanlı oldukça hızlı bir modern site oluşturucusudur.
 
-## Get Started
+## Başlarken
 
-There are two main ways to get started with Gatsby:
+Gatsby'ye başlamanın 2 yolu var:
 
-1. [Tutorial](/tutorial/): Step-by-step instructions on how to install Gatsby and start a project: written for people without Gatsby or web development experience, though these learning resources have helped developers of all skill levels.
-2. [Quick start](/docs/quick-start): One page summary of how to install Gatsby and start a new project.
+1. [Eğitim](/tutorial/): Adım adım talimatları izleyerek sıfırdan Gatsby kurulumu yaparak bir proje oluşturun: Daha önce hiç Gatsby ya da web geliştirme deneyimi olmayan, ancak her beceri seviyesindeki yazılımcılara da yardımcı olur.
```

Comment:
We should keep using `-` instead of `+`. It can break markdown files.

Reply:
I think you are talking about the diff's indicators. That minus/plus isn't releated with markdown.

Reply:
You are right. My bad.